src/test/java/com/neo4j/docker/TestExtendedConf.java

```diff
@@ -98,7 +105,78 @@ void testReadsTheExtendedConfFile_defaultUser(String password) throws Exception
 		}
 	}
 
+	@ParameterizedTest
+	@ValueSource( strings = {"", "secretN30"} )
+	void testReadsExistingConfFileWithExtension_rootUser( String password ) throws Exception
+	{
+		// set up test folders
+		Path testOutputFolder = HostFileSystemOperations.createTempFolder( "extendedConfIsRead-" );
+		Path logsFolder = HostFileSystemOperations.createTempFolder( "logs-", testOutputFolder );
 
+		try ( GenericContainer container = createContainer( password ) )
+		{
+			HostFileSystemOperations.mountHostFolderAsVolume( container, logsFolder, "/logs" );
+			container.start();
+			Container.ExecResult result = container.execInContainer(
+					"bash", "-c", "printf '\\ndbms.logs.http.rotation.keep_number=$(expr 2 * 10)\\n' >> ${NEO4J_HOME}/conf/neo4j.conf"
+			);
+			String logs = String.format( "stdout:\n%s\n\nstderr:\n%s", result.getStdout(), result.getStderr() );
+			Assertions.assertEquals( 0, result.getExitCode(), logs );
+			restartNeo4j( container );
+
+			runContainerAndVerify( container, null, logsFolder, password );
+		}
+	}
+
+	@ParameterizedTest
+	@ValueSource( strings = {"", "secretN30"} )
+	void testReadsExistingConfFileWithExtension_7474User( String password ) throws Exception
+	{
+		// set up test folders
+		Path testOutputFolder = HostFileSystemOperations.createTempFolder( "extendedConfIsRead-" );
+		Path logsFolder = HostFileSystemOperations.createTempFolder( "logs-", testOutputFolder );
+
+		try ( GenericContainer container = createContainer( password ) )
+		{
+			container.withCreateContainerCmdModifier( (Consumer<CreateContainerCmd>) cmd -> cmd.withUser( "7474:7474" ) );
```

Comment:
I forgot why this is necessary. Is it something to do with the entrypoint needing to run as 7474 user?

Reply:
we use 7474:7474 as the default non-root user in K8s (running as root is bad etc.)

This is checking that if that user is used and `EXTENDED_CONF` is set then things should work (even if no external neo4j.conf has been provided into `/config/`)